src/govuk/components/skip-link/skip-link.test.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env jest */
+
+const configPaths = require('../../../../config/paths.json')
+const PORT = configPaths.ports.test
+
+const baseUrl = 'http://localhost:' + PORT
+
+describe('/examples/template-default', () => {
+  describe('skip link', () => {
+    beforeAll(async () => {
+      await page.goto(`${baseUrl}/examples/template-default`, { waitUntil: 'load' })
+      await page.keyboard.press('Tab')
+      await page.click('.govuk-skip-link')
+    })
+
+    it('focuses the target element', async () => {
+      const activeElement = await page.evaluate(() => document.activeElement.id)
+
+      expect(activeElement).toBe('main-content')
+    })
+
+    it('adds the tabindex attribute to the target', async () => {
+      const tabindex = await page.$eval('.govuk-main-wrapper', el => el.getAttribute('tabindex'))
+
+      expect(tabindex).toBe('-1')
+    })
+
+    it('adds the class for removing the native focus style to the target', async () => {
+      const cssClass = await page.$eval('.govuk-main-wrapper', el => el.getAttribute('class'))
+
+      expect(cssClass).toContain('govuk-skip-link-focused-element')
```

Comment:
I'd suggest using `classlist.contains` here, because it's a slightly more robust check.

For example, this would currently pass if there was a class that contained this string  `class="govuk-skip-link-focused-element-foo"` or if there was a missing space between two class names (`class="another-classgovuk-skip-link-focused-element"`)

```suggestion
      const hasClass = await page.$eval('.govuk-main-wrapper',
        el => el.classList.contains('govuk-skip-link-focused-element')
      )

      expect(hasClass).toBeTruthy()
```